src/BinaryTree/EnumerableExtensions.cs

```diff
@@ -7,10 +7,100 @@ public static class EnumerableExtensions
     {
         public static void PrintToConsole<T>(this IEnumerable<T> source)
         {
-            foreach (var item in source)
+            if (source is BinaryTree<int>)
```

Comment:
I think it should be another method or method with parameter, because sometimes I need to print it as a tree and sometimes as plain text